feeds/wifi-ax/hostapd/files/hostapd.sh

```diff
@@ -214,6 +214,9 @@ hostapd_common_add_bss_config() {
 	config_add_int maxassoc max_inactivity
 	config_add_boolean disassoc_low_ack isolate short_preamble
 
+config_add_int signal_connect signal_stay signal_poll_time \
```

Comment:
Ran into an issue on the wifi-6 ap's where we weren't applying the configuration for `signal_stay` and some other related fields to the hostapd configuration file. Making the client_disconnect_threshold field not work for wifi-6 aps.